### PR TITLE
Improve Lisk Service setup guide - Closes #1446

### LINKED ISSUE #1446: Improve Lisk Service docs
### Description

- remove the ‘export’ keyword in the example for ‘.env’ file (Installation with Docker). The quotes could also be removed.
- consider adding the complete command line flags to start lisk-core without passing the config file as well. The documentation assumes that ‘/home/lisk/lisk-core/’ exists, which is not always the case.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1446

### CODE REVIEW

**Review by chris529 — DISMISSED** [review]
LGTM! 👍

File: modules/ROOT/pages/setup/docker.adoc [hunk 1]
```diff
@@ -44,49 +44,21 @@ Check out the xref:{url_core_setup_npm}[Lisk Core NPM setup] guide for more info
...
-<1> Allows remote servers to connect to the Lisk Core WS API.
-If this option is not set, it defaults to `127.0.0.1`.
-
 Use the Lisk Core CLI to start Lisk Core with the desired configurations.
 
-The `--network` flag determines the blockchain network Lisk Core connects to.
-
-`--api-ws` enables the WebSocket API of the node, if not already enabled in the config.json file.
+* The `--network` flag determines the blockchain network Lisk Core connects to.
+* `--api-ws` enables the WebSocket API of the node, if not already enabled in the config.json file.
```
Comment (sameersubudhi) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
* `--api-ws` enables the WebSocket API of the node, if not already enabled within the config.json file.
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 2]
```diff
@@ -44,49 +44,21 @@ Check out the xref:{url_core_setup_npm}[Lisk Core NPM setup] guide for more info
...
+If this option is not set, it defaults to `127.0.0.1`.
+* `--api-ws-port=8080` the port for the WebSocket API.
 
 [source,bash]
 ----
-lisk-core start --network mainnet --api-ws --config=/home/lisk/lisk-core/custom-config.json --overwrite-config
+lisk-core start --network mainnet --api-ws --api-ws-host=0.0.0.0 --api-ws-port=8080
 ----
 
 To connect to a different network, replace `mainnet` with either `testnet` or `devnet`.
```
Comment (sameersubudhi) [inline modules/ROOT/pages/setup/docker.adoc:60]:
```suggestion
To connect to a different network, replace `mainnet` with `testnet`, `alphanet` or `betanet`. To start your own private network for development purposes use `devnet`.
```

**Review by sameersubudhi — DISMISSED** [review]
LGTM 👍 
Please find below a couple of optional suggestions.

File: modules/ROOT/pages/setup/docker.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/setup/docker.adoc:60]: do we want to publicly document alphanet and betanet?

Comment (sameersubudhi) [reply modules/ROOT/pages/setup/docker.adoc:60]: We can leave it as is if we do not specify them in any other product documentation. Please feel free to resolve the comment.

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: modules/ROOT/pages/setup/docker.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/setup/docker.adoc:60]:
```suggestion
To connect to the public test network, replace `mainnet` with `testnet` or, to start your own private network for development purposes, use `devnet`.
```

**Pull request merged**